utils/ipfs.ts

```diff
@@ -31,21 +20,3 @@ export const extractCid = (ipfsLink?: string): string => {
 
   return match[1]
 }
-
-const IPFS2AR = 'https://ipfs2arweave.com/permapin/'
-export const ipfsToArweave = async (ipfsLink: string): Promise<string> => {
```

Comment:
🤔 this is not part of minimark afaik

Reply:
Yes, but it also has never been used/called.

Reply:
So I remove it.